src/express/utils/create-lambda-request-handler.ts

```diff
@@ -30,8 +30,9 @@ export function createLambdaRequestHandler(
           event: {
             ...getRequestHeaders(req),
             path: req.path,
+            httpMethod: req.method,
             queryStringParameters: req.query,
-            body: req.body
+            body: req.body ? JSON.stringify(req.body) : null
```

Comment:
Is this a bug in the current implementation? After a short research the body in the event of a Lambda proxy integration seems to be indeed a string.

@unstubbable Can you confirm that?

Reply:
Since the text parser is used, req.body is already a string. Did this not work in your case?

Reply:
I was experiencing `body` being passed to lambda as empty object. But looking at the type of `APIGatewayProxyEvent` it should only be `string | null`.

Reply:
An additional Stringify may even lead to an invalid result. What would be if I get a JSON string which I then stringify again. I would then have double quotation marks. Wouldn't it be better to check if it is already a string, if not then use Stringify or null.

Reply:
Thats a very valid point. I didn't realised that `JSON.stringify` adds additional quotes. I changed it to be used for non-strings only.